arches_her/pkg/graphs/resource_models/Artefact.json

```diff
@@ -21711,6 +21718,11 @@
                     "alias": "shine_form",
                     "config": {
                         "options": [
+                            {
+                                "id": "e8d04473-da05-4cda-95f4-ebfd0ff28122",
+                                "selected": false,
+                                "text": "N/A"
+                            },
```

Comment:
@SDScandrettKint - the addition N/A to the "Shine form" node is unexpected as it is a concept and the collection does not have any of the other High-Low items in. The PR is for a change to domain options only.

Can you double check this and the other models?

Reply:
Removed all options from shine_form leaving just the rdm reference in [34068a3](https://github.com/archesproject/arches-her/pull/1230/commits/34068a3815902b90a568cf3da089fc7bd40bb5d7).

Also ensured that N/A is always before Low in the order.
(Note that the order of options is "High" -> "N/A" in all files apart from in branches/Location Data where the order is the reverse "N/A" -> "High")